x-pack/legacy/plugins/task_manager/task.ts

```diff
@@ -209,6 +209,11 @@ export interface TaskInstance {
    * and then query such tasks to provide a glimpse at only reporting tasks, rather than at all tasks.
    */
   scope?: string[];
+
+  /**
+   * The random uuid of the Kibana instance which claimed ownership of the task last
+   */
```

Comment:
nit: but could we do `ownerId` as it's more descriptive of what this is?

Reply:
Yeah, sure. :)